dask_jobqueue/jobqueue.yaml

```diff
@@ -34,6 +35,7 @@ jobqueue:
     local-directory: null       # Location of fast local storage like /scratch or $TMPDIR
 
     # PBS resource manager options
+    shebang: "#!/bin/bash"
```

Comment:
Shouldn't we go for `#!/usr/bin/env bash` ?

Reply:
Yep, looks like it, good catch!